README.md

```diff
@@ -166,18 +166,18 @@ an excelent toolkit for information retrieval research.
 First, we need to install Maven, and clone and compile Anserini's repository:
 ```
 sudo apt-get install maven
-git clone https://github.com/castorini/Anserini.git
+git clone --recurse-submodules https://github.com/castorini/Anserini.git
```

Comment:
the recurse-submodules flag will install anserini tools along with the anserini repository